.github/workflows/windows.yml

```diff
@@ -143,6 +159,16 @@ jobs:
       #
       # Upload build artifacts
       #
+
+      - name: Pack Artifacts
```

Comment:
lets do not additionally pack it, it is small

Reply:
We need it to get correct Windows archive to upload